renderer/main.js

```diff
@@ -109,7 +109,7 @@ function onState (err, _state) {
 
   // Log uncaught JS errors
   window.addEventListener('error',
-    (e) => telemetry.logUncaughtError('window', e.error), true)
+    (e) => telemetry.logUncaughtError('window', e.error || e.target), true)
```

Comment:
Shouldn't an ErrorEvent always have `e.error`?
https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent

What kinds of errors are there where we need to use target instead?